play-java-grpc-example/project/plugins.sbt

```diff
@@ -1,7 +1,12 @@
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.0")
+enablePlugins(BuildInfoPlugin)
+val playGrpcV = "0.8.1"
+buildInfoKeys := Seq[BuildInfoKey]("playGrpcVersion" -> playGrpcV)
```

Comment:
The gRPC version is used in the plugin file and in the project itself. To avoid adding the version in many places, I'm adding it on BuildInfo and reusing it.